pyproject.toml

```
@@ -1,170 +1,155 @@
-[project]
+[tool.poetry]
 name = "education-backend"
 version = "2022.10.31"
```

```suggestion
version = "2023.09.18"
```

Давай бампнем уже, а то мы делаем довольно важное изменение.